### PR TITLE
Check if request object has the query_params attr

### PR DESCRIPTION
**Pull request by sbchisholm**
Suggested fix for #2

### CODE REVIEW

**Comment by dbrgn** [comment]
What DRF version are you using? In my case `query_params`  seems to exist even if I don't pass in any GET parameters.

Do you know the reason for the missing `query_params` attribute?

**Comment by sbchisholm** [comment]
I'm not sure if you saw issue #2 I mentioned that it's only happening when running our unit tests.  We're using drf's `APIRequestFactory` to generate request objects, this is returning a [WSGIRequest](https://github.com/django/django/blob/master/django/core/handlers/wsgi.py#L77) which does not have the attribute `query_params`.  Using `drf-extensions==0.2.8` and `Django==1.9.8`.

In this case it appears that the `query_params` show up under the attribute `GET`.

File: drf_dynamic_fields/__init__.py [hunk 1]
```diff
@@ -18,6 +18,10 @@ def __init__(self, *args, **kwargs):
             warnings.warn('Context does not have access to request')
             return
 
+        # If the request object does not have query_params attribute, return
+        if not hasattr(self.context['request'], 'query_params'):
+            return
+
```
Comment (sbchisholm) [inline drf_dynamic_fields/__init__.py]: @dbrgn perhaps the better change would be to get fields from either the `query_params` or the `GET` attribute.

**Comment by dbrgn** [comment]
Fine, thanks :)

**Comment by dbrgn** [comment]
Version 0.1.1 is released.

**Pull request merged**